app/src/main/java/com/github/studydistractor/sdp/data/UserData.kt

```diff
@@ -11,11 +11,13 @@ package com.github.studydistractor.sdp.data
  * @constructor create a user
  */
 data class UserData(
-    var id: String? = null,
+    var id: userId? = null,
```

Comment:
Okay but then why only this one and not eventId and messageId and the others

Reply:
(Personally i think this should just be string)

Reply:
I did it for the list of participants which makes it clearer that we have a list of `userId`, but I agree that this is not necessary

Reply:
Maybe choose either one way or the other :) I don't have a preference.

Reply:
Alright! Let's do string everywhere :)